scripts/migrate_to_fork.py

```diff
@@ -339,7 +339,7 @@ def setup_remotes(username: str, fork_url: str, auto_accept: bool = False) -> st
     # Handle upstream remote
     upstream_remote = None
     for name, url in remotes.items():
-        if 'leanprover-community/mathlib4' in url:
+        if url == 'https://github.com/leanprover-community/mathlib4.git' or url == 'git@github.com:leanprover-community/mathlib4.git':
```

Comment:
This might be too strict, I have the remote at "https://github.com/leanprover-community/mathlib4". Can we do `endswith "leanprover-community/mathlib4.git" or endswith "leanprover-community/mathlib4"` instead?

Reply:
Sure, I can add a few more cases, but `endswith` might be too flexible (I'd rather not have to deal with problems when someone creates an account named `cleanprover-community` or something like that). How about this?
```suggestion
        if re.fullmatch(r'.*[/:]leanprover-community/mathlib4(\.git)?', url):
```